CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 * ...
 
+## 2.1.4 2022-07-05
+
+* Truncate: Simplify VALUES(...) lists
+* Truncate: Correctly handle UPDATE and ON CONFLICT target lists
+* Support complex queries with deeply nested ASTs ([#238](https://github.com/pganalyze/pg_query/pull/238))
```

Comment:
@Tassosb We should also add the following two that were added after this PR was opened:

```
* Find table references inside type casts
* Find function calls referenced in expression indexes
```